websocket/connection.go

```diff
@@ -165,7 +166,7 @@ func NewDurableConnection(target string, messageChan chan []byte, logger *zap.Su
 				}
 				logger.Debug("Connected to ", target)
 				if err := c.keepalive(); err != nil {
-					logger.With(zap.Error(err)).Errorf("Connection to %s broke down, reconnecting...", target)
+					logger.Errorw(fmt.Sprintf("Connection to %s broke down, reconnecting...", target), zap.Error(err))
```

Comment:
nit, saves one string parsing if I'm not mistaken (although to be fair I've never any difference comparing both in benchmarks).
```suggestion
					logger.Errorw(fmt.Sprint("Connection to ", target, " broke down, reconnecting..."), zap.Error(err))
```

Reply:
Yeaaaaah, well, I'll stick the current version then, if there's no significant diff anyway.

Reply:
The reason I commented is because @vagababov consistently asked me to not use `Sprintf` whenever possible in the past, and he's assigned to this PR.

Reply:
That is indeed a different convo and our linter currently only catches trailing transforms.

I do think we should disable that linter if it does not result in a measurable performance benefit (which AFAIK it doesn't, at least not in the cases I checked) as it just makes code-writing (especially logging) feel more foreign then it should.

I'm all for good code-style etc. but IMO this is just overly verbose (and I have to remember which functions add spaces between the arguments and which don't).